test/transactsql.test.ts

```diff
@@ -118,27 +118,37 @@ describe('TransactSqlFormatter', () => {
     `);
   });
 
-  it('formats SET NOCOUNT ON; GO CREATE OR ALTER TABLE', () => {
-    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER TABLE t');
+  it('formats SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE', () => {
+    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE p');
     expect(result).toBe(dedent`
     SET
     NOCOUNT ON;
 
     GO
-    CREATE OR ALTER TABLE
-      t
+    CREATE OR ALTER PROCEDURE
+      p
     `);
   });
 
-  it('formats SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE', () => {
-    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE p');
+  it('formats SELECT ... INTO clause', () => {
+    const result = format('SELECT col INTO #temp FROM tbl');
     expect(result).toBe(dedent`
-    SET
-    NOCOUNT ON;
+    SELECT
+      col
+    INTO
+      #temp
+    FROM
+      tbl
```

Comment:
Should indent the expected formatted code by extra 2 spaces, like done in other tests.

Reply:
Confirming the indentation has been updated